app/Http/Controllers/Users/UsersController.php

```diff
@@ -639,6 +639,27 @@ public function emailAssetList($id)
         $user->notify((new CurrentInventory($user)));
         return redirect()->back()->with('success', trans('admin/users/general.user_notified'));
     }
+    /**
+     * Emails All users a list of assigned assets
+     *
+     * @author [G. Martinez] [<godmartinz@gmail.com>]
+     * @since [v6.0.10]
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function emailEveryoneAssetList()
+    {
+        $this->authorize('view', User::class);
+
+        $users= User::all()->where('email')->whereNotNull();
```

Comment:
We should probably not send emails to users who do not have anything assigned to them.

Reply:
This now finds the users with assets (`findAssetOwners()`) and emails only these users.